### PR TITLE
Self managed onpremises

### PR DESCRIPTION
**Pull request by bmckercher123**
New How to for Get Started rework

- Can merge at any time 
- Assign to Casey for final review
- Reviewed by Spike 
- Part of Alex reorg work: Alex provided inputs on scope of doc

### CODE REVIEW

File: _data/master/navbars/getting-started.yml [hunk 1]
```diff
@@ -20,20 +22,6 @@ toc:
...
-    - title: Installing Calico for policy and networking (recommended)
-      path: /getting-started/kubernetes/installation/calico
-    - title: Installing Calico for policy (advanced)
-      path: /getting-started/kubernetes/installation/other
-    - title: Enabling application layer policy for Istio
-      path: /getting-started/kubernetes/installation/app-layer-policy
-    - title: Customizing the manifests
-      path: /getting-started/kubernetes/installation/config-options
-    - title: Integration guide (advanced)
-      path: /getting-started/kubernetes/installation/integration
```
Comment (spikecurtis) [inline _data/master/navbars/getting-started.yml]: Why have all these links been removed?  It looks like `getting-started/kubernetes/installation/other.md` was removed, but all the rest of the docs still exist.

File: _data/master/navbars/reference.yml [hunk 2]
```diff
@@ -150,6 +150,8 @@ toc:
         path: /reference/architecture/design/l2-interconnect-fabric
       - title: Calico over IP fabrics
         path: /reference/architecture/design/l3-interconnect-fabric
+- title: Customize Calico manifests
+  path: /getting-started/kubernetes/installation/config-options
```
Comment (spikecurtis) [inline _data/master/navbars/reference.yml]: this path is incorrect because the file was renamed

File: calico.yaml [hunk 3]
```diff
@@ -0,0 +1,796 @@
+---
```
Comment (spikecurtis) [inline calico.yaml]: Was this file added by mistake?

File: _data/master/navbars/getting-started.yml (hunk 1 above)
Comment (bmckercher123) [reply _data/master/navbars/getting-started.yml]: Customizing the manifest is moved to Reference (per Casey decision)
Integration Guide is removed -- it was replaced by Calico the hard way and was never removed (per Casey).

**Pull request closed without merging**